### PR TITLE
Fix style of /wiki/Registration

### PR DESCRIPTION
**Pull request by miterosan**

### CODE REVIEW

File: wiki/Registration/en.md [hunk 1]
```diff
@@ -1,43 +1,39 @@
...
-| b. Alphanumeric characters ("a-z, A-Z, 0-9"), underscores ("\_"), squared brackets, ("\[" and "\]") and dashes ("-") are the only characters allowed. |
-| **2. E-mail Address**                                                                                                                                 |
-| **3. Password**                                                                                                                                       |
-| **4. Note**                                                                                                                                           |
-| **5. Continue button**                                                                                                                                |
+1. On the [main webpage](https://osu.ppy.sh), click ``I'm new!`` to start the registration process.
+![Register-3.png](Register-1.png)
+2. Fill out all fields in the form below and press ``Continue``  
+![Login.png](Register-2.png)
+ 1. Username
```
Comment (MegaApplePi) [inline wiki/Registration/en.md]: add one more space before the `1.`

File: wiki/Registration/en.md [hunk 2]
```diff
@@ -1,43 +1,39 @@
...
-| **2. E-mail Address**                                                                                                                                 |
-| **3. Password**                                                                                                                                       |
-| **4. Note**                                                                                                                                           |
-| **5. Continue button**                                                                                                                                |
+1. On the [main webpage](https://osu.ppy.sh), click ``I'm new!`` to start the registration process.
+![Register-3.png](Register-1.png)
+2. Fill out all fields in the form below and press ``Continue``  
+![Login.png](Register-2.png)
+ 1. Username          
+  1. Maximum of 15 characters
```
Comment (MegaApplePi) [inline wiki/Registration/en.md]: add two more spaces before the `1.`

File: wiki/Registration/en.md [hunk 3]
```diff
@@ -1,43 +1,39 @@
...
-| **3. Password**                                                                                                                                       |
-| **4. Note**                                                                                                                                           |
-| **5. Continue button**                                                                                                                                |
+1. On the [main webpage](https://osu.ppy.sh), click ``I'm new!`` to start the registration process.
+![Register-3.png](Register-1.png)
+2. Fill out all fields in the form below and press ``Continue``  
+![Login.png](Register-2.png)
+ 1. Username          
+  1. Maximum of 15 characters                                                                                                                    
+  2. Alphanumeric characters (``a-z``, ``A-Z``, ``0-9``), underscores (``_``), squared brackets, (``[`` and ``]``) and dashes (``-``) are the only characters allowed.
```
Comment (MegaApplePi) [inline wiki/Registration/en.md]: add two more spaces before the `2.`

File: wiki/Registration/en.md [hunk 4]
```diff
@@ -1,43 +1,39 @@
...
+ 2. E-mail Address
```
Comment (MegaApplePi) [inline wiki/Registration/en.md]: add one more space before the `2.`

File: wiki/Registration/en.md [hunk 5]
```diff
@@ -1,43 +1,39 @@
...
-| **5. Continue button**                                                                                                                                |
+1. On the [main webpage](https://osu.ppy.sh), click ``I'm new!`` to start the registration process.
+![Register-3.png](Register-1.png)
+2. Fill out all fields in the form below and press ``Continue``  
+![Login.png](Register-2.png)
+ 1. Username          
+  1. Maximum of 15 characters                                                                                                                    
+  2. Alphanumeric characters (``a-z``, ``A-Z``, ``0-9``), underscores (``_``), squared brackets, (``[`` and ``]``) and dashes (``-``) are the only characters allowed.
+ 2. E-mail Address                                                                                                                                                     
+ 3. Password
```
Comment (MegaApplePi) [inline wiki/Registration/en.md]: add one more space before the `3.`

File: wiki/Registration/en.md [hunk 6]
```diff
@@ -1,43 +1,39 @@
...
+1. On the [main webpage](https://osu.ppy.sh), click ``I'm new!`` to start the registration process.
+![Register-3.png](Register-1.png)
+2. Fill out all fields in the form below and press ``Continue``  
+![Login.png](Register-2.png)
+ 1. Username          
+  1. Maximum of 15 characters                                                                                                                    
+  2. Alphanumeric characters (``a-z``, ``A-Z``, ``0-9``), underscores (``_``), squared brackets, (``[`` and ``]``) and dashes (``-``) are the only characters allowed.
+ 2. E-mail Address                                                                                                                                                     
+ 3. Password                                                                                                                                                            
+ 4. Note
```
Comment (MegaApplePi) [inline wiki/Registration/en.md]: add one more space before the `4.`

File: wiki/Registration/en.md [hunk 7]
```diff
@@ -1,43 +1,39 @@
...
+![Register-3.png](Register-1.png)
+2. Fill out all fields in the form below and press ``Continue``  
+![Login.png](Register-2.png)
+ 1. Username          
+  1. Maximum of 15 characters                                                                                                                    
+  2. Alphanumeric characters (``a-z``, ``A-Z``, ``0-9``), underscores (``_``), squared brackets, (``[`` and ``]``) and dashes (``-``) are the only characters allowed.
+ 2. E-mail Address                                                                                                                                                     
+ 3. Password                                                                                                                                                            
+ 4. Note                                                                                                                                                               
+ 5. Continue button
```
Comment (MegaApplePi) [inline wiki/Registration/en.md]: add one more space before the `5.`

File: wiki/Registration/en.md [hunk 8]
```diff
@@ -1,43 +1,39 @@
...
-| **1. osu! executable** |
-| **2. Options panel**   |
-| **3. User panel**      |
 
-</center>
 What's next?
 ------------
 
-You're ready to start your rhythm adventure, so proceed to the [welcome](welcome "wikilink") page and start playing! Don't forget to read the [rules](rules "wikilink") very carefully!
+You're ready to start your rhythm adventure, so proceed to the [welcome](/wiki/Welcome/) page and start playing! Don't forget to read the [rules](/wiki/Rules/) very carefully!
```
Comment (MegaApplePi) [inline wiki/Registration/en.md]: ASG says, "no contractions" (this is to keep the writing a bit more formal);
- Change `You're` to `You are`
- Change `Don't` to `Do not`

File: wiki/Registration/en.md [hunk 9]
```diff
@@ -1,43 +1,39 @@
 How do I register?
 ------------------
 
-<center>
-**<font size="3">Please note that having more than one account at any time is an infringement of <span class="plainlinks"> \[</wiki/Game_Modes/osu!>!:Rules osu! rules\]</span>. Follow the below steps to set up your osu! account.**
-</font>
-<span style="font-size:18px">1. On the <span class="plainlinks"> [main webpage](https://osu.ppy.sh)</span>, click "<b>I'm new!</b>" to start the registration process. </span>
 
-<span style="font-size:18px">2. Fill out all fields in the form below and press "Continue" </span>
+**Please note that having more than one account at any time is an infringement of [Rules osu! rules](/wiki/Rules/). Follow the below steps to set up your osu! account.**
```
Comment (MegaApplePi) [inline wiki/Registration/en.md]: `of Rules osu! rules`... wth?

---

Replace line with: `**Please note that having more than one account at any time is an infringement of the [osu! rules](/wiki/Rules/). Follow the below steps to set up your osu! account.**`

File: wiki/Registration/en.md [hunk 10]
```diff
@@ -1,43 +1,39 @@
...
-| **1. osu! executable** |
-| **2. Options panel**   |
-| **3. User panel**      |
 
-</center>
 What's next?
 ------------
 
-You're ready to start your rhythm adventure, so proceed to the [welcome](welcome "wikilink") page and start playing! Don't forget to read the [rules](rules "wikilink") very carefully!
+You are ready to start your rhythm adventure, so proceed to the [welcome](/wiki/Welcome/) page and start playing! Don't forget to read the [rules](/wiki/Rules/) very carefully!
```
Comment (MegaApplePi) [inline wiki/Registration/en.md]: Change `Don't` to `Do not`

File: wiki/Registration/en.md [hunk 11]
```diff
@@ -1,43 +1,39 @@
...
+| osu! executable | ![Register.png](Register-3.png) | This executable (which should be named osu!.exe) will open the game itself. If updates are required, they will be applied automatically. _Note: You do not need to open osu! as an administrator. If the game requires it, it will prompt for it itself._                                                |
+| Options panel   | ![Login.png](Login.png)         | If this is your first time starting osu!, you will see this on the left side of your game's screen. Enter your username and password to their respective fields and log in.                                                                                                                              |
+| User panel      | ![Userpanel.png](Userpanel.png) | Once you've entered your username and password correctly, you will see your user panel at the top left of your screen. You are now logged into osu! and ready to play! You will also see your user panel on the bottom of your song selection screen, to the right of your ``Selection Options`` button. |
```
Comment (MegaApplePi) [inline wiki/Registration/en.md]: change `you've` to `you have`

**Comment by Repflez** [comment]
Is this ready for merge?

**Comment by miterosan** [comment]
yes

**Pull request merged**